pubspec.yaml

```
@@ -10,29 +10,34 @@ environment:
   sdk: '>=2.18.4 <3.0.0'
 
 dependencies:
+  cupertino_icons: ^1.0.2
   flutter:
     sdk: flutter
 
+  flutter_svg: ^1.1.6
+  simple_logger: ^1.9.0+1
 
-  cupertino_icons: ^1.0.2
+  yumemi_weather:
+    git:
+      url: https://github.com/yumemi-inc/flutter-training.git
+      ref: main
+      path: packages/yumemi_weather
 
 dev_dependencies:
-  flutter_test:
-    sdk: flutter
-
-  flutter_lints: ^2.0.0
   blendthink_lints: ^3.0.0
-
+  flutter_lints: ^2.0.0
+  flutter_test: 
+    sdk: flutter
 
 # The following section is specific to Flutter packages.
 flutter:
 
   uses-material-design: true
 
-  # To add assets to your application, add an assets section, like this:
-  # assets:
-  #   - images/a_dot_burr.jpeg
-  #   - images/a_dot_ham.jpeg
+  assets:
+    - assets/images/cloudy.svg
+    - assets/images/rainy.svg
+    - assets/images/sunny.svg
```

[imo]
ここは個人的には保守性を考えると、ディレクトリで指定してしまったほうがいいかもしれません！
```suggestion
    - assets/images/
```

---

なるほどです！
参考になります🙇